api/management-model/build.gradle.kts

```diff
@@ -54,12 +57,37 @@ openApiGenerate {
   serverVariables = mapOf("basePath" to "api/v1")
 }
 
+sourceSets {
+    main {
```

Comment:
Please run `./gradlew spotlessApply` before pushing to PRs :) CI failed because of that.